usng4j-api/src/main/java/org/codice/usng4j/api/CoordinateSystemTranslator.java

```diff
@@ -21,7 +21,7 @@
  * SOFTWARE.
  **/
 
-package org.codice.usng;
+package org.codice.usng4j.api;
```

Comment:
You shouldn't need to put the interfaces in a `api` package.  Putting the implementations in a `impl` package is enough to avoid the split package issue.